Makefile.am

```diff
@@ -62,6 +62,7 @@ noinst_LTLIBRARIES += libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.
 
 libeos_updater_util_libeos_updater_util_@EUU_API_VERSION@_la_CFLAGS = \
 	$(AM_CFLAGS) \
+	$(STD_CFLAGS) \
```

Comment:
Would it make sense to do `AM_CFLAGS = $(STD_CFLAGS) $(CODE_COVERAGE_CFLAGS) $(WARN_CFLAGS)` to reduce clutter?

Reply:
Yes and no. It would reduce clutter, but I find it makes it harder to work out which set of flags apply to a particular target, hence I prefer to be more explicit about it (unless dealing with a lot of very similar targets, like test binaries, where we can guarantee more uniformity). It is a bit of personal preference though.